src/components/button-minor/button-minor.pw.tsx

```diff
@@ -0,0 +1,644 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import { ButtonMinorProps } from "./button-minor.component";
+import {
+  Default as ButtonMinor,
+  ButtonMinorCustom,
+  ButtonMinorDifferentTypes,
+} from "./components.test-pw";
+import {
+  PrimaryButton,
+  PrimaryDestructiveButton,
+  PrimaryDisabledButton,
+  PrimaryIconButton,
+  PrimaryFullWidthButton,
+  PrimaryNoWrapButton,
+  SecondaryButton,
+  SecondaryDestructiveButton,
+  SecondaryDisabledButton,
+  SecondaryIconButton,
+  SecondaryFullWidthButton,
+  SecondaryNoWrapButton,
+  TertiaryButton,
+  TertiaryDestructiveButton,
+  TertiaryDisabledButton,
+  TertiaryIconButton,
+  TertiaryFullWidthButton,
+  TertiaryNoWrapButton,
+  IconOnlyButton,
+  IconOnlyWithTooltipButton,
+} from "./button-minor.stories";
+import { buttonMinorComponent } from "../../../playwright/components/button/index";
+import { BUTTON_ICON_POSITIONS } from "../button/button.config";
+import { ICON } from "../../../playwright/components/locators";
+import {
+  dlsRoot,
+  icon,
+  tooltipPreview,
+} from "../../../playwright/components/index";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import {
+  assertCssValueIsApproximately,
+  checkAccessibility,
+} from "../../../playwright/support/helper";
+import { HooksConfig } from "../../../playwright";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+
+const buttonBackgrounds = [
```

Comment:
**suggestion:** May need to change the name of this const as it seems to relate to the position of the button not the background of the button.